cnd/src/http_api.rs

```diff
@@ -93,17 +93,17 @@ impl From<(tables::Order, tables::BtcDaiOrder)> for OrderProperties {
             price: Amount::from(btc_dai_order.price),
             quantity: Amount::from(btc_dai_order.quantity),
             state: State::new(
-                order.open,
-                order.closed,
-                order.settling,
-                order.failed,
-                order.cancelled,
+                Amount::from(btc_dai_order.open),
```

Comment:
How is `OrderProperties` only ever instantiated for the DB? How is it saved to the DB in the first place?

Reply:
@da-kami can you answer the question? I am curious :)